app/views/handlers/view_submitted_response_pdf.py

```diff
@@ -54,7 +54,8 @@ def __init__(
     @property
     def filename(self) -> str:
         """The name to use for the PDF file"""
-        return f"{self._metadata['schema_name']}.pdf"
+        if self._questionnaire_store.submitted_at is not None:
```

Comment:
```suggestion
        if self._questionnaire_store.submitted_at:
```
but can it not have `submitted_at`?